website/docs/_posts/configuration-options.md

```diff
@@ -908,15 +908,6 @@ Whether to convert ranged versions in `package.json` to pinned versions.
 
 This is a very important feature to consider, because not every repository's requirements are the same. The default value within the tool itself is false, which means no existing ranges are pinned. However if you are using the suggested preset `"config:base"`, then it changes the default of pinVersions to `null`, which means Renovate attempts to autodetect what's best for the project. In such cases `devDependencies` in `package.json` will alway be pinned, but `dependencies` will only be pinned if the package is `private` or has no `main` entry defined - both indicators that it is not intended to be published and consumed by other packages. To override the `"config:base"` setting, add the preset `":preserveSemverRanges"` to your `extends` array.
 
-## pip_requirements
```

Comment:
why is this removed?

Reply:
It will be added back again after merging with master.

Reply:
But is this just because your branch/fork is out of date with the `master` of this repo? I much prefer you update your fork than to merge and have it out of date immediately